src/Components/Request/VisibleElements.vue

```diff
@@ -14,7 +14,7 @@
 			</h2>
 			<p>
 				<small>
-					{{ t('libresign', 'Select each signer to define their signature positions') }}
+					{{ t('libresign', 'Click the page to sign, then select a signer to set their signature position') }}
```

Comment:
What you think about this suggestion?

```suggestion
					{{ t('libresign', 'Click the page to add visible signature, then select a signer to set their signature position') }}
```

Is possible to sign without visible signature. If you don't want to add visible signature, you only need to ignore this step and go ahead.